tsconfig.json

```diff
@@ -3,8 +3,8 @@
         "outDir": "dist",
         "strict": true,
         "module": "commonjs",
```

Comment:
This is not good, but our imports of apollo break thanks to their broken types: https://arethetypeswrong.github.io/?p=%40apollo%2Fclient%403.9.11

Reply:
Odd, since they _use_ @arethetypeswrong/cli